helm-chart/values.yaml

```diff
@@ -1,11 +1,28 @@
 ---
-releaseImageTag: cf66cfb # Need to pass this value when applying the helm chart (This is the $BUILD_ID )
+# Need to pass $BUILD_ID to this key when applying the helm chart in CI. 
+# or 
+# Set the commit hash for a particular buiild to deploy built image for that commit ID. (NOTE: This would work only if the image is built for that commit ID.)
+releaseImageTag: cf66cfb 
```

Comment:
this shouldn't be defaulted, or default to latest if they have those tags

Reply:
If we want to manage a distribution of releases to consumers, then it should be defaulted to the tagged images associated with that release?

Reply:
Removed the default value out of the values file for now. Will update the manifests to use `imageRegistry` and `tag` as two separate inputs for the image.